gradle/maven.gradle

```diff
@@ -9,6 +9,15 @@
  *
  */
 plugins.withId('maven') {
+    task sourcesJar(type: Jar, dependsOn: classes) {
+         classifier = 'sources'
+         from sourceSets.main.allSource
+    }
+    artifacts{
```

Comment:
Isn't this going to publish ALL test JARs, instead of just the one we want?

Reply:
Yes Andrei that is intentional and it is not visible unless users click on 'view all' as on this page: https://repo1.maven.org/maven2/io/pravega/pravega-test-testcommon/0.7.1